packages/app/components/empty-placeholder.tsx

```diff
@@ -30,7 +32,9 @@ const EmptyPlaceholder: React.FC<EmptyPlaceholderProps> = ({
 
   return (
     <View tw={`items-center justify-center pt-4 ${tw}`}>
-      <Text tw="text-lg font-extrabold text-gray-900 dark:text-gray-100">
+      <Text
+        tw={`text-lg font-extrabold text-gray-900 dark:text-gray-100 ${titleTw}`}
```

Comment:
i think capsize settings might not be working well for emoticons. Sometimes it trims the top so adding padding here.